packages/hydrogen/src/UIMsgButton.tsx

```diff
@@ -379,7 +379,7 @@ export const UIMsgButton = ({
                     {iconPosition === UIMsgButtonIconPosition.Left && icon != null && (
                         <Button.Icon
                             source={icon}
-                            style={!!title ? styles.leftIcon : null}
+                            style={title ? styles.leftIcon : null}
```

Comment:
Maybe there was a case for the check? 🤔

Reply:
It's OK since it's not `!!title && styles.leftIcon` :)